core/shared/src/main/scala/zio/InMemoryCache.scala

```diff
@@ -0,0 +1,49 @@
+package zio
```

Comment:
Make sure you copy the licence from some other file.

Reply:
Yeah, sorry, I'm used to `sbt-header` based "auto-headering", so I never really think about these kinds of things anymore.
Should ZIO maybe rely on sbt-header as well?